packages/x-charts/src/internals/plugins/featurePlugins/useChartCartesianAxis/zoom.types.ts

```diff
@@ -99,10 +99,16 @@ export interface ZoomSliderOptions {
    * @default 'hover'
    */
   showTooltip?: ZoomSliderShowTooltip;
+  /**
+   * If `true`, a preview of the chart will be shown in the slider.
+   */
+  preview?: boolean;
```

Comment:
```suggestion
  preview?: boolean;
```

I think the next steps would be to allow different types of preview, rather than just the same type of chart as the main chart.

Eg: for a scatter chart, you might want to display a line representing the y values aggregated by some rule (every 1min for example if you are displaying fine-grained time series).

This would probably require some configuration, so we might want to review this to `boolean | { ...config }` eventually